### ISSUE TITLE
Calling smart contract Update() method from invoke file gets VM error

### ISSUE THREAD

**Issue by roschler** [neo-project/neo-express#158]
When I call my smart contract *Update()* method from an invoke file, I get this VM fault:
```

EXECUTION:
#1

TRIGGER:
Application

VM STATE:
FAULT

EXCEPTION:
Object reference not set to an instance of an object.

GAS:
1914120
```
Anybody know how to diagnose this?

When I debug this step from an *invoke-file*, I see this error in the DEBUG CONSOLE:
```
Engine State Fault: Exception has been thrown by the target of an invocation. [TargetInvocationException]
  Contract Exception: Wrong magic [FormatException]
```

I am running the invoke call from the contract owner account and choosing the _CalledByEntry_ execution mode.

**Comment by roschler** [comment]
Just a follow-up.  I went ahead and completely reset the blockchain from within the N3 plugin.  In other words, I didn't use my build command or NEO express batch commands at all.  From within the N3 plugin I also transferred some gas from genesis to owner, did a deploy from the plugin interface, and then finally called my invoke file that calls the transfer method on the GasToken contract.  Everything worked and I did see the message from my `OnNEP17Transfer()` method.  It looks like something I'm doing from the Terminal via the neoxp batch commands is throwing things out of sync.  In fact, I am wondering if the **magic number** is different between the instance I work with when running commands directly from the Terminal window and that the N3 plugin is using when calling invoke-files.
